src/utils/EPToolkit.ts

```diff
@@ -90,10 +92,13 @@ export function exchange_text(text: string, options: IOptions): Buffer {
   bytes.concat(init_printer_bytes);
 
   // set encoding
-  if (m_options["encoding"] && options_controller["encoding"][m_options["encoding"]]) {
+  if (
```

Comment:
You  have a lot of code formatting changes which are not consistent across this repository